js/utils/autoUpdate.js

```diff
@@ -0,0 +1,60 @@
+import $ from 'jquery';
+import { ipcRenderer } from 'electron';
+import app from '../app';
+import Dialog from '../views/modals/Dialog';
+
+let statusMsg;
+let removeStatusMsgTimout;
+
+export function showUpdateStatus(status = '', msg = '', type = 'message') {
```

Comment:
Is it really necessary to have both the `status` and `msg` variables? It seems `msg` is never being passed in and status is always being passed in as a sentence (with a period), so it would never make sense to concatenate the two with a colon.